CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 ## Unreleased
   - Fix custom info label field in forms LEKP-rapport - Melding correctie authentieke bron and LEKP-rapport - Toelichting Lokaal Bestuur (DL-5934)
+  - Add new report `emailsInFailboxWithin24Hours` which tracks failed emails (DL-5943)
```

Comment:
Maybe consider renaming the function to something a bit more generic. It might be in the end we want 36 hours or last month or.